test/suite/test_compact10.py

```diff
@@ -140,12 +140,12 @@ def test_compact10_full_backup(self):
         assert self.get_files_compacted(uris) == self.num_tables
 
         # Take a second full backup.
-        os.mkdir(self.backup_dir_2)
-        self.take_full_backup(self.backup_dir_2)
+        os.mkdir(self.backup_incr)
+        self.take_full_backup(self.backup_incr)
 
         # Compare the backups.
         for uri in uris:
-            self.compare_backups(uri, self.backup_dir_1, self.backup_dir_2)
+            self.compare_backups(uri, self.backup_full, self.backup_incr)
```

Comment:
Same here, we are comparing two full backups, not a full and an incremental one.

Reply:
I have changed the naming [3570009](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/3570009210cdd4c711eea7aae6427731f792f903)